substrate/client/rpc-api/src/state/mod.rs

```diff
@@ -38,6 +38,13 @@ pub trait StateApi<Hash> {
 	#[method(name = "state_call", aliases = ["state_callAt"], blocking)]
 	fn call(&self, name: String, bytes: Bytes, hash: Option<Hash>) -> Result<Bytes, Error>;
 
+	/// Debug a method from the runtime API.
+	/// The method will be called at the block's parent state.
+	/// and passed the block to be replayed, followed by the decoded `bytes`
+	/// arguments.
+	#[method(name = "state_debugBlock", blocking, with_extensions)]
+	fn debug_block(&self, name: String, block: Hash, bytes: Bytes) -> Result<Bytes, Error>;
```

Comment:
I don't get why you should pass the `name: String` here? If this RPC is only to call `ReviveApi_trace_tx`, it should hardcode it?

Reply:
also asking better docs as to what each param is in general, RPC docs are important and get propagated into PJS/PAPI.

Reply:
ah maybe this is not the one that is meant to be used by end users?

Reply:
+1 for the docs will add more details

> I don't get why you should pass the name: String here?

A few reason, We have 2 runtime methods that make use of this. One that replay an entire block, the other that replay a single transaction. Replaying a single transaction involve that you replay all the transactions that precede 
to ensure that it's replayed from the same state that when it was executed.

Also potentially other Runtime API could make use of this, so hard coding it does not  make much sense here.